mongo/options/clientoptions_test.go

```diff
@@ -240,6 +240,60 @@ func TestClientOptions(t *testing.T) {
 			})
 		}
 	})
+	t.Run("heartbeatFrequencyMS validation", func(t *testing.T) {
```

Comment:
Optional: Move these test cases into a separate top-level test to make them easier to read and run.

E.g.
```go
func TestClientOptions_HeartbeatFrequencyMS(t *testing.T) {
	// ...
```

Reply:
I'm opting to not move them into a separate test to maintain conformity with other client options!